tests/integration/frameworks/drupal/mock_module_handler_empty_array.php

```diff
@@ -0,0 +1,34 @@
+<?php
+/*
+ * Copyright 2020 New Relic Corporation. All rights reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/* Verify agent behavior when key value is an empty array */
+
+namespace Drupal\Core\Extension {
+  interface ModuleHandlerInterface
+  {
+    public function invokeAllWith($hook_str, $callback);
+  }
+  class ModuleHandler implements ModuleHandlerInterface
+  {
+    protected array $hookImplementationsMap = array(
```

Comment:
In the test run actions, the PHP 7.2 and 7.3 test runs are failing due to some PHP language change around this.  Since this is only mocking 11.1 structure anyway, those PHPs can be excluded in a skipif.

Reply:
typed properties weren't introduced until [7.4](https://php.watch/versions/7.4/typed-properties)

fixed here 2481b5be246166c69a9c80ac97899151ee9b39fb

Reply:
Technically we could set the skipif to 8.3+ since that's all that 11.1+ supports. I'm torn- I want to make sure we aren't impacting the prior Drupal hook instrumentation methodology, but this map won't exist for less than PHP 8.3

Reply:
No strong opinions on this one.

For the mocked versions, either way is fine.  It's okay to leave for php 7.4+ since it's just a mock and doesn't really run, as you mentioned it just verifies that the structure parsing works and handles various valid/invalid values.

And for multiverse, you'll already be excluding PHPs based on actual drupal code and the PHPs it supports, so it should be fine.